lisa/runners/tests/test_lisa_runner.py

```diff
@@ -51,6 +52,9 @@ def generate_runner(
 class RunnerTestCase(TestCase):
     __skipped_no_env = "no available environment"
 
+    def setUp(self) -> None:
```

Comment:
time to fix these camel cases here, no?

Reply:
It's from builtin TestCase class of Python. see https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp